docs/variables.md

```diff
@@ -810,9 +865,13 @@ Maximum allowed value is `2000000` (two million delete markers).
 | Data type    | Boolean                                          |
 | Default      | OFF                                              |
 
-Specifies whether to count single deletes as delete markers
-recognized by rocksdb_compaction_sequential_deletes.
-Disabled by default.
+!!! note
+
+    In version [Percona Server for MySQL 8.0.36-28](.//release-notes/8.0.36-28.md) and later, the default value is changed from `OFF` to `ON`.
+
+The default value is `ON` which means the variable is enabled.
```

Comment:
Usual pattern is definition of the variable and then the default value.

Reply:
fixed